cmd/podman/system_prune.go

```diff
@@ -0,0 +1,63 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/containers/libpod/libpod/adapter"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	"github.com/urfave/cli"
+)
+
+var (
+	pruneSystemDescription = `
+	podman system prune
+
+        Remove unused data
+`
+	pruneSystemFlags = []cli.Flag{
+		cli.BoolFlag{
+			Name:  "all, a",
+			Usage: "remove all unused data",
+		},
+		cli.BoolFlag{
+			Name:  "force, f",
+			Usage: "Do not prompt for confirmation",
+		},
+	}
+	pruneSystemCommand = cli.Command{
+		Name:         "prune",
+		Usage:        "Remove unused data",
+		Description:  pruneSystemDescription,
+		Action:       pruneSystemCmd,
+		OnUsageError: usageErrorHandler,
+		Flags:        pruneSystemFlags,
+	}
+)
+
+func pruneSystemCmd(c *cli.Context) error {
+
+	runtime, err := adapter.GetRuntime(c)
+	if err != nil {
+		return errors.Wrapf(err, "could not get runtime")
+	}
+	defer runtime.Shutdown(false)
+	// Call prune; if any cids are returned, print them and then
+	// return err in case an error also came up
+	pruneCids, err := runtime.PruneImages(c.Bool("all"))
+	if len(pruneCids) > 0 {
+		for _, cid := range pruneCids {
+			fmt.Println(cid)
+		}
+	}
+	err1 := volumePrune(runtime, getContext(), c.Bool("force"))
```

Comment:
I don't think `docker system prune` actually prunes volumes. The description says exclusively containers, images, networks

Reply:
It does, see https://docs.docker.com/engine/reference/commandline/system_prune/

Reply:
There's a `--volume` flag.